publish.sh

```
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+# Wrapper script for backward compatibility
+# Use tasks.sh directly for more options: ./tasks.sh --help
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)